tests/scripts/test_psa_compliance.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""Run the PSA Crypto API compliance test suite.
+
+Clone the repo and check out the commit specified by PSA_ARCH_TEST_REPO and PSA_ARCH_TEST_REF,
```

Comment:
Nit: `PSA_ARCH_TEST_REPO` is no longer defined here so it's a bit weird to reference it in this script's docstring.